src/style.css

```diff
@@ -1 +1,30 @@
-/* add styles here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
```

Comment:
Avoid using the `*` selector for setting styles as it impacts performance. It is better to set styles only for elements that require them. You can create a separate class for these properties and apply to the necessary elements.